external/docker-integration-tests/src/test/scala/org/apache/spark/sql/jdbc/MySQLIntegrationSuite.scala

```diff
@@ -39,6 +39,8 @@ class MySQLIntegrationSuite extends DockerJDBCIntegrationSuite {
   }
 
   override def dataPreparation(conn: Connection): Unit = {
+    // Since MySQL 5.7.14+, we need to disable strict mode
```

Comment:
just out of curiosity; why we need to disable the mode in this mysql?

Reply:
As I mentioned in the PR description, MySQL baned `0000-00-00 00:00:00`. There is a SQL mode for that. I enumerate all versions and tested. It starts to ban on 5.7.14+.

Reply:
oh, I missed that. thanks.